src/renderer/viz/expressions/ramp.js

```diff
@@ -1,7 +1,33 @@
 import BaseExpression from './base';
 import { implicitCast, checkLooseType, checkExpression, checkType, clamp, checkInstance } from './utils';
-import { cielabToSRGB, sRGBToCielab } from '../colorspaces';
+import { interpolate } from '../colorspaces';
 import Sprites from './sprites';
+import NamedColor from './color/NamedColor';
+import Buckets from './buckets';
+import Property from './basic/property';
+import { Classifier } from './classifier';
+import Linear from './linear';
+import Top from './top';
+
+const paletteTypes = {
```

Comment:
ping :slightly_smiling_face:

Reply:
I think it is more readable, at list for this long class, to have a place to look when you want to know which are the different palette types available. Is not a Java-JavaScript-OtherLanguage issue, but it is a good practice to avoid both strings and magic numbers when possible. I also thing this constants should be in a separate file along with other constants so we can have them available when needed, but that is another discussion 🙈